project-reports/2023/2023-Q2-Hyperledger-Aries.md

```diff
@@ -0,0 +1,211 @@
+of existing deployments to [Aries Askar], the use of ACA-Py
+as an Aries Mediator (the [Aries Mediator Service] repository), updating
+ACA-Py to use Hyperledger AnonCreds, and the launch of a new ACA-Py
+documentation site [aca-py.org]
+
+[aca-py.org]: https://aca-py.org/
+[Aries Mediator Service]: https://github.com/hyperledger/aries-mediator-service
+
+The Indy/Aries stack continues to be the global leader in SSI/verifiable
+data solutions, with AnonCreds the most used credential format.
+
+# Questions/Issues for the TOC
+
+None.
+
+# Releases
+
+The following Aries releases occurred in the last quarter:
+
+-   Aries Cloud Agent Python -- 0.8.0-rc0, 0.8.0
+-   Aries Framework JavaScript -- v0.3.2, v0.3.3, 0.3.4-alpha0 to 0.3.4-alpha17, 0.4.0-alpha18 to 0.4.0-alpha95
+-   Aries Askar -- v0.2.8
+-   Aries VCX Releases -- v0.3.4, v0.4.0-dev1 to v0.4.0-dev12
+-   Aries Framework Go -- v0.1.9
+
+Of the 28 Aries repositories, all but 3 have been updated in 2023.
+
+Interoperability status can be seen
+here: [https://aries-interop.info](https://aries-interop.info).
+
+# Overall Activity in the Past Quarter
+
+Per the [Aries Activity Dashboard](https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Faries/dashboard;subTab=technical?time=%7B%22from%22:%222023-01-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-03-31T07:00:00.000Z%22%7D) for the first three months of 2023, Aries
+codebases had 442 PRs (up 11%) from 81 contributors (up 21%).
+
+# Current Plans
+
+-   Continuing the work on eliminating the use of the Indy SDK in favor of the
+    Aries Askar/Indy VDR/anoncreds-rs across frameworks and deployments of those
+    frameworks.
+-   Continued expansion of features in mobile wallet apps, including building on
+    the wallet-to-wallet support added in Q1, and expanding support for OCA to
+    enable beautifully displayed credentials.
+-   Continue the work to adding ledger agnostic Hyperledger AnonCreds support in
+    all of the active Aries Frameworks.
+-   Promotion of a place for issuers to publish and wallets/verifiers to use
+    Overlays Capture Architecture (OCA) bundles.
+-   Rust [DID resolver, DID parser](https://github.com/hyperledger/aries-vcx/pull/812) in Aries VCX
+    - Not specific to Aries
+    - Currently implementing did:sov only, did:indy, did:web will be added next
+
+# Maintainer Diversity
+
+There are currently 62 people on the 48 Hyperledger Aries teams representing at least 19 organizations.
+
+# Contributor Diversity
+
+In Q1 there were contributions from 81 individuals representing at least 16 organizations.
+
+# Additional Information
+
+Nothing
+
+# Submission date
+
+03-May-2023
+
+# Reviewed by
+
+-   🔲 Arnaud J Le Hors
+-   🔲 Arun S M
+-   🔲 Bobbi Muscara
+-   🔲 David Enyeart
+-   🔲 Jim Zhang
+-   🔲 Marcus Brandenburger
+-   🔲 Peter Somogyvari
+-   🔲 Ramakrishna V
+-   🔲 Stephen Curran
+-   🔲 Timo Glastra
+-   🔲 Tracy Kuhrt
```

Comment:
```suggestion
```

These are no longer needed know that we have moved to GitHub. Information captured in this PR.